src/stateManager/StateManager.ts

```diff
@@ -308,104 +312,102 @@ class StateManager {
         onChainTimestamp?: Timestamp,
         validationStrategy?: AValidationStrategy
     ): Promise<boolean> {
-        // the try/catch is to ensure that the mutex is unlocked in case of an error
-        // no error is actually expected to happen, and the catch block just re-throws the error
-        const strategy =
-            validationStrategy || this.getStrategyByStatus(this.status);
         try {
-            await this.mutex.lock();
-            let validationResult: BlockValidationResult =
-                BlockValidationResult.SUCCESS;
-            const isAuthentic =
-                await this.diamondStateMachine.localDiamondContract.isBlockAuthentic(
-                    blockConfirmation.signedBlock
-                );
-
-            if (!isAuthentic) {
-                validationResult =
-                    await strategy.authenticateBlockFailed(blockConfirmation);
-                return await strategy.interpretFinalValidationResult(
-                    validationResult
-                );
-            }
+            // the try/catch is to ensure that the mutex is unlocked in case of an error
+            // no error is actually expected to happen, and the catch block just re-throws the error
+            const strategy =
+                validationStrategy || this.getStrategyByStatus(this.status);
+            try {
+                await this.mutex.lock();
+                let validationResult: BlockValidationResult =
+                    BlockValidationResult.SUCCESS;
+                const isAuthentic =
+                    await this.diamondStateMachine.localDiamondContract.isBlockAuthentic(
+                        blockConfirmation.signedBlock
+                    );
 
-            const block = Block.fromBlockConfirmation(
-                blockConfirmation,
-                onChainTimestamp
-            );
+                if (!isAuthentic) {
+                    validationResult =
+                        await strategy.authenticateBlockFailed(
+                            blockConfirmation
```

Comment:
this just has different formatting - not sure is it from eslint or prettier - prettier works for me, so I guess it's from eslint, otherwise it would have the same format?